algrtm/src/components/PianoRollDescription.js

```diff
@@ -0,0 +1,34 @@
+import Image from "next/image";
+import React from "react";
+
+const PianoRollDescription = () => {
+  return (
+    <section className="flex flex-col items-center justify-center w-full px-[12.62rem] mt-[3.94rem]">
+      <div className="flex gap-[4.5rem] items-center w-full">
```

Comment:
In Figma, the entire section looks centered, so adding `justify-center` and removing `w-full` will change the width to the same as in Figma, and also center this div